keanu-project/src/main/java/io/improbable/keanu/vertices/intgr/probabilistic/MultinomialVertex.java

```diff
@@ -39,7 +39,7 @@ public MultinomialVertex(IntegerVertex n, DoubleVertex p) {
     }
 
     public MultinomialVertex(int n, DoubleVertex p) {
-        this(ConstantVertex.of(n), p);
+        this(ConstantVertex.of(IntegerTensor.create(n, new long[]{1, 1})), p);
```

Comment:
elsewhere you've used `IntegerTensor.scalar(n).reshape(1, 1)`. Do you think it's worth standardising, and/or introducing a helper method `IntegerTensor.scalar1by1(n)` and similarly for `DoubleTensor`?

Reply:
It might be worth adding a helper method for it but I didn't think it should go in this PR.

Reply:
ok